RGPosadas/LitterRally
# TASK-46: Refresh Access Token after expiration

This PR closes #170.

**Steps to test**
1. Start the application using `nx serve mull-ui` & `nx serve mull-api`.
2. Delete the `mullToken` cookie from your browser if it is present.
3.  Head to the `auth.service.ts` and change the expiration time of `createAccessToken` to `15s` or any other duration to make testing easier.
4. Open the application with the developer tools on the network tab.
5. When making a request after the access token expires, there should be a refresh request.
6. There should be no indication that something went wrong and the request should work as intended.
